### DISCUSSION TITLE
No response from support in 30 days, URGENT PLEASE!

### DISCUSSION THREAD

**Discussion by RenatoDev3** [vercel/community#4773] [Account Issue]
### Question

I tried creating an account on Vercel over a month ago and got a message saying "This user account is blocked." and contacted support, haven't gotten a response in over a month.

Account registration shouldn't be this difficult, here is my case number, please resolve the issue. 

Case number: 00165149

Thank you!

**Comment by amyegan** [comment]
It looks like your case received a response a couple of weeks ago. We can't safely discuss account details in public, so please follow up with your support case 🙏

**Reply by RenatoDev3** [reply]
Amy, I got a response a couple of weeks ago saying that my account was flagged for additional review during the registration process and is currently blocked which is completely fine. I then got asked if I could share details about the projects I plan on hosting on Vercel.

I simply responded that I plan on hosting my portfolio projects. It has been more than 2 weeks from that point!

So please tell me, how should I follow up with my support case, I already answered and have been waiting for more than 2 weeks.
